builtin/mainmenu/dlg_config_world.lua

```diff
@@ -16,6 +16,9 @@
 --51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 --------------------------------------------------------------------------------
+local function modname_valid(name)
+	return string.len(string.gsub(name, "[abcdefghijklmnopqrstuvwxyz0123456789_]*", "")) == 0
+end
```

Comment:
* This should reference the place in the core where this is checked and vice-versa, since they both have to be changed together.
  * IIRC capital letters are allowed too.
  * A much cleaner and faster way to do this:
```Lua
return not name:find("[^a-zA-Z0-9_]")
```